app/assets/javascripts/application.js

```diff
@@ -28,5 +28,27 @@
 $(function () {
   $('[data-toggle="tooltip"]').tooltip()
 
-  new Tablesort(document.getElementById('analytics-table'), { descending: true })
+  if($('#analytics-table').length) {
```

Comment:
I left a comment about this JS in the other [PR](https://github.com/simpledotorg/simple-server/pull/1189)

Reply:
Thanks! I'll move non-reports related conditionals/selectors to JS view files in a separate PR